lib/modules/blockchain_process/blockchain.js

```diff
@@ -64,7 +64,7 @@ var Blockchain = function(options) {
     this.config.account = {};
     this.config.account.password = fs.embarkPath("templates/boilerplate/config/development/password");
     this.config.genesisBlock = fs.embarkPath("templates/boilerplate/config/development/genesis.json");
-    this.config.datadir = fs.embarkPath(".embark/development/datadir");
+    this.config.datadir = fs.dappPath(".embark/development/datadir");
```

Comment:
What do you think of that change?
That was setting the directory to embark installation
I believe it is wrong but pls confirm

Reply:
Yeah, you're right. It must have been a copy/paste, because the other ones are ok.